docs/data/material/guides/nextjs/nextjs.md

```diff
@@ -149,3 +178,30 @@ If you are using TypeScript, add `DocumentHeadTagsProps` to the Document's props
    ...
  }
 ```
+
+### Link
+
+For pages router, use the Link only from `@mui/material-nextjs/*-pagesRouter`:
+
+```js
+import { Link } from '@mui/material-nextjs/v13-pagesRouter'; // or `v14-pagesRouter` if you are using Next.js v14
```

Comment:
I think we should also document how to use a Link with a Button.

```jsx
import Button from '@mui/material/Button';
import NextLink from 'next/link';

<Button component={NextLink} href="/foo">
```

Actually, I don't know if this works (maybe types issue, maybe href prop conflict) but this is definitely an important use case.

Reply:
Also it should be possible to pass in `NextLink` via a theme, right? Should we have that example as well?